sale_configurator_base/views/sale_view.xml

```diff
@@ -101,7 +104,12 @@
                     name="open_sale_line_config_base"
                     type="object"
                     icon="fa-th"
-                    attrs="{'invisible': ['|', ('state', 'not in', ('draft', 'sent')), ('is_configurable', '=', False)]}"
+                    attrs="{'invisible': ['|', ('state', 'not in', ('draft', 'sent')), '|', ('is_configurable', '=', False), ('is_configured', '=', False)]}"
+                />
+                <button
+                    title="This product was not added as configurable"
+                    icon="fa-ban"
+                    attrs="{'invisible': ['|', ('state', 'not in', ('draft', 'sent')), '|', ('is_configurable', '=', False), ('is_configured', '=', True)]}"
```

Comment:
If  configurable product  was added  without using configuring interface, we cant edit it to add option ?